packages/presets/src/fragments/outline/outline-viewer.ts

```diff
@@ -20,9 +17,6 @@ import {
 
 export const AFFINE_OUTLINE_VIEWER = 'affine-outline-viewer';
 
-@requiredProperties({
-  editor: PropTypes.instanceOf(AffineEditorContainer),
-})
 @customElement(AFFINE_OUTLINE_VIEWER)
```

Comment:
Instead of remove this, can we replace with some other check?

Reply:
After discussing with @akumatus , we can use `PropTypes.object` to cover null and undefined check which is the most common case. 

```ts
@requiredProperties({
  editor: PropTypes.object,
})
```